### PR TITLE
[Doc Improvement][Hyperlink updated][3249282]

### PR DESCRIPTION
**Pull request by v-rmvv**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e9f23083fdd37b539115df56b281bae65000d056/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37736c83-7dbd-414c-a32a-5fcc250432b0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e7bb84f93ee29129a9e1865770c8c0d139dacdfa/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a43b479-5358-44cb-9d82-45defd4fd307) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9af040417dd5e12b2b3ccc4e499b5f741a0e23ef/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4734940a-4c14-41e9-bf4f-4307a4f8f201) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/60cc9218657206ef532cbb860f9157fe94e28e71/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 68 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb7a5e9c-179c-44a2-a82e-e1748cfab9de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c8e545ca940193223d2db2a62b3eef3fe439006/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 68 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8840af36-4046-4fe7-8eb8-feccc3a94637) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/authentication/add-authentication.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ In this article you'll learn:
     | Sample | BotBuilder version | Demonstrates |
     |:---|:---:|:---|
     | **Bot authentication** in [cs-auth-sample][teams-auth-bot-cs] | v4 | OAuthCard support |
-    | **Bot authentication** in [js-auth-sample][teams-auth-bot-js] | v4| OAuthCard support  |
+    | **Bot authentication** in [js-auth-sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-teams-authentication/nodejs) | v4 | OAuthCard support |
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/authentication/add-authentication.md:36]: Please don't directly add the link here:

There will be a section for `[teams-auth-bot-js]` so please add the link there or please create another section if duplicate.

File: msteams-platform/bots/how-to/authentication/add-authentication.md [hunk 2]
```diff
@@ -280,9 +280,9 @@ With the preliminary settings done, let's focus on the creation of the bot to us
 
 # [JavaScript](#tab/node-js)
 
-1. Clone [node-auth-sample][teams-auth-bot-js].
+1. Clone [node-auth-sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-teams-authentication/nodejs).
```
Comment (ChetanSharma-msft) [inline msteams-platform/bots/how-to/authentication/add-authentication.md:283]: Please don't directly add the link here:

There will be a section for `[teams-auth-bot-js]` so please add the link there or please create another section if duplicate.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c54ede8f4e17fd95dfbb51a09b40787cd3ee5b3/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 68 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ea06e742-3152-49ac-b7ad-d7c6c5229896) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/authentication/add-authentication.md (hunk 1 above)
Comment (v-ganr) [reply msteams-platform/bots/how-to/authentication/add-authentication.md]: Updated.

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine.

**Review by surbhigupta12 — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/182efc54f81244dded7c063633bc84d3d9971881/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 68 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ad38079e-56e7-4b76-af30-8d5bb31383df) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a6f5e1b982ed8faecf36bfcaa0c080c1d66d96af/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fadd-authentication.md) | 90 | 68 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eccef64b-77bf-4a1f-b71c-92b71a792c4c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**